packages/wrangler/src/dev.tsx

```diff
@@ -61,7 +62,7 @@ function Dev(props: DevProps): JSX.Element {
   // kinda forbid that, so we thread the entry through useCustomBuild
   const entry = useCustomBuild(props.entry, props.buildCommand);
 
-  const format = useWorkerFormat({ file: entry, format: props.format });
+  const format = useWorkerFormat({ file: entry?.file, format: props.format });
```

Comment:
I think we need to pass the whole `entry` to `useWorkerFormat()` since it will try to build that file using esbuild, and the working directory is relevant.

Reply:
nice catch, thanks!